src/components/MaterialUI/DataDisplay/PredefinedElements/StepperModal.js

```diff
@@ -77,7 +77,15 @@ const useStyles = makeStyles(theme => ({
 	},
 }));
 
-const StepperModal = ({ steps = [], title = "", open, closeCallback, confirmCallback, confirmTitle }) => {
+const StepperModal = ({
+	steps = [],
+	title = "",
+	open,
+	closeCallback,
+	confirmCallback,
+	confirmTitle,
+	type = "normal",
```

Comment:
Default should be wide

Reply:
That will break Settings... they are using it for ManageRoleListModal component.

Reply:
Changed to wide.